client/src/graphql/queries/admin.graphql

```diff
@@ -2,6 +2,7 @@ query GetSystemConfig {
   systemConfig {
     motd
     maxFileUploadSize
+    defaultIdeDockerImage
```

Comment:
I think we should either use
`ideDockerImage` and `defaultIdeDockerImage`
or
`ideImage` and `defaultIdeImage`